README.md

```diff
@@ -1,11 +1,29 @@
 # cheri-examples
 cheri-riscv sample c programs
 
+------
 ## Build instructions with CMake
+Build al the example code with CMake using  default settings :
 1. Create a separate build directory (\<build-dir\>) outside the source directory (\<source-dir\>).
-2. Generate the build files using either **Unix Makefiles** or **Ninja**. Currently only the *RISC-V* cheri-purecap toolchain file has been created. From outside the source directory execute: 
+2. Generate the build files using either **Unix Makefiles** (*default*) or **Ninja** (specfied
+   with **-G**). Currently only the *RISC-V* cheri-purecap toolchain file has been created.
+   1. If executing from inside the build directory :
 
-   `cmake -B <build-dir> -S <source-dir> -DCMAKE_TOOLCHAIN_FILE=<source-dir>/riscv64-purecap.cmake -DSDK=<cheri-sdk-dir> -G Ninja`
-3. Build all the examples 
+   `cmake -DCMAKE_TOOLCHAIN_FILE=riscv64-purecap.cmake <relative-path-to-source-dir>`
 
-   `cmake --build <build-dir>`
+   2. If executing from outside the build directory :
```

Comment:
Will anyone ever want to build from outside the build directory? This feels like it might come under the "if you want to do this, you know what to do anyway" heading?

Reply:
Keeps it consistent with make/ninja rule generation instructions. Let me know if not necessary and I can remove it

Reply:
I have no preferences here, as I don't use systems like ninja. So if you think that's best, let's go with it.